src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -731,6 +733,16 @@ You can set the [aggregation window duration](/docs/alerts-applied-intelligence/
   Baseline alert condition thresholds don't support editing the aggregation window. They use the 1 minute default.
 </Callout>
 
+### Sliding window aggregation
```

Comment:
@paperclypse Can you add `[#sliding-window-aggregation]` here because i'd like to link to this spot in [this ticket](https://newrelic.atlassian.net/browse/AINTER-8220) for the "See our docs" link in the UI.  Thanks!